nuxeo-core/nuxeo-core-bulk/src/main/java/org/nuxeo/ecm/core/bulk/io/BulkJsonWriter.java

```diff
@@ -48,8 +48,8 @@ public BulkJsonWriter() {
     @Override
     public void writeEntityBody(BulkStatus entity, JsonGenerator jg) throws IOException {
         jg.writeStringField(BULK_ID, entity.getId());
-        jg.writeStringField(BULK_STATE, entity.getState().toString());
-        jg.writeStringField(BULK_SUBMIT, entity.getSubmitTime().toString());
+        jg.writeStringField(BULK_STATE, entity.getState() != null ? entity.getState().toString() : null);
```

Comment:
Maybe it's worth adding next to `AbstractJsonWriter#writeSerializableField` a `writeOptionalSerializableField` wdyt ?

Reply:
It could be worth, but it would imply to add more logic on how to serialize some objects ( an `Instant` for example)